tools/depends/configure.ac

```diff
@@ -270,6 +285,7 @@ case $host in
     optimize_flags="-O2"
     platform_cxxflags="$platform_cflags"
     platform_os="linux"
+    ffmpeg_options="--with-vaapi --with-vdpau"
```

Comment:
At tools/depends/target/ffmpeg/Makefile they are listed as `--enable-vdpau --enable-vaapi`. Is the `--with` syntax also correct?
As you add it here it should be removed at the other file.

Reply:
thx